server/connection/socket.js

```diff
@@ -0,0 +1,93 @@
+const Websocket = require('ws');
+const {
+  createChat,
+  getReciveChatRoomInfo,
+  plusSellerUnreadCount,
+  plusBuyerUnreadCount,
+  updateLastChat,
+} = require('../data/chat');
+
+const rooms = new Map();
+
+class Socket {
+  constructor(server) {
+    this.wss = new Websocket.Server({
+      server,
+      verifyClient: async (info, cb) => {
+        const parsedUrl = info.req.url.split('/');
+        const [productId, buyerId, sellerId] = [parsedUrl[2], parsedUrl[3], parsedUrl[4]];
+        const roomId = parseInt(String(productId) + String(buyerId));
```

Comment:
확실하진 않지만 요렇게 할수도 있을 거예요
```suggestion
        const roomId = `${productId}${buyerId}` * 1;
```

Reply:
이런 방법도 있군요... 대단하세요..